### PR TITLE
Pallet lifter for fast battery swapping.

### PR DESCRIPTION
**Pull request by Vasyan2006**
Add makeshift pallet lifter vehicle part.
After installing on shopping cart frame it will have lifting and jacking qualities 1 (up to 500 kg or 1100 lbs) and will allow to swap storage batteries.
Recipe requires fabrication 2 to unlock and bottle jack with pipes to craft.
Compare to existing cranes it is easier to craft but provides less utility.

### CODE REVIEW

File: data/json/items/vehicle/engineering.json [hunk 1]
```diff
@@ -24,5 +24,18 @@
     "material": "steel",
     "symbol": "&",
     "color": "dark_gray"
+  },
+  {
+    "id": "pallet_lifter",
+    "type": "GENERIC",
+    "category": "veh_parts",
+    "name": "pallet lifter",
+    "description": "A makeshift pallet lifter. If attached to a frame it could be used to lift up to 500 kg.",
```
Comment (Night-Pryanik) [inline data/json/items/vehicle/engineering.json]: Double space after `lifter.` please.

Comment (Vasyan2006) [reply data/json/items/vehicle/engineering.json]: Done.

File: data/json/items/vehicle/engineering.json [hunk 2]
```diff
@@ -24,5 +24,18 @@
     "material": "steel",
     "symbol": "&",
     "color": "dark_gray"
+  },
+  {
+    "id": "pallet_lifter",
+    "type": "GENERIC",
+    "category": "veh_parts",
+    "name": "pallet lifter",
+    "description": "A makeshift pallet lifter.  If attached to a frame it could be used to lift up to 500 kg.",
```
Comment (DracoGriffin) [inline data/json/items/vehicle/engineering.json]: Suggest changing `500 kg` to `0.5 metric tonnes` to keep in line with other lifters.

Comment (Vasyan2006) [reply data/json/items/vehicle/engineering.json]: Done.

**Pull request merged**